lib/controller.js

```diff
@@ -1026,6 +1026,17 @@ controller.tesselEnvVersions = opts => {
   });
 };
 
+controller.reboot = opts => {
+  opts.authorized = true;
+  return controller.standardTesselCommand(opts, tessel => {
+    return tessel.reboot().then(() => {
+      log.info('Tessel Rebooted');
```

Comment:
Things get racy here...
- I'd say _still rebooting_ (will reword output)
- Yes `reboot` and power cycle are different.

A power cycle is SoC and Coprocessor clearing their states. `reboot` only resets the SoC's state. See [reset button](https://github.com/tessel/t2-firmware/issues/185#issuecomment-244577542) issue.